qiskit/providers/ibmq/job/queueinfo.py

```diff
@@ -56,5 +62,66 @@ def __init__(
         self.hub_priority = hub_priority
         self.group_priority = group_priority
         self.project_priority = project_priority
+        self.job_id = job_id
 
         super().__init__(**kwargs)
+
+    def __repr__(self) -> str:
+        """Return the official string representation of QueueInfo.
+
+        Returns:
+            a string representation of QueueInfo.
+        """
+        status = api_status_to_job_status(ApiJobStatus(self._status)).value \
+            if self._status else self._get_value(self._status)
+        estimated_start_time = self.estimated_start_time.isoformat() \
+            if self.estimated_start_time else self._get_value(self.estimated_start_time)
+        estimated_complete_time = self.estimated_complete_time.isoformat() \
+            if self.estimated_complete_time else self._get_value(self.estimated_complete_time)
+
+        queue_info = [
+            "job_id='{}'".format(self._get_value(self.job_id)),
+            "_status='{}'".format(self._get_value(status)),
+            "estimated_start_time='{}'".format(estimated_start_time),
+            "estimated_complete_time='{}'".format(estimated_complete_time),
+            "position='{}'".format(self._get_value(self.position)),
+            "hub_priority='{}'".format(self._get_value(self.hub_priority)),
+            "group_priority='{}'".format(self._get_value(self.group_priority)),
+            "project_priority='{}'".format(self._get_value(self.project_priority))
```

Comment:
nit - you don't actually need to quote `position` and `priority`'s, since they're not strings.